e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -426,6 +426,13 @@ Scenario('local authority enters attending hearing',async (I, caseViewPage, ente
 
 Scenario('local authority uploads documents', caseDocs.uploadDocuments());
 
+Scenario('local authority cannot upload court bundle', async (I, caseViewPage, uploadDocumentsEventPage) => {
+  await caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  I.dontSee(uploadDocumentsEventPage.documents.courtBundle);
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+});
```

Comment:
this seems like new functionality 🤔

Reply:
The merge combined 2 events that had slightly different functionality, one had the ability to upload court bundles and the other didn't. I had a chat with Moray and he said that even when the events have been merged that the solicitor shouldn't have that ability to upload a court document before it is submitted.